packages/Webkul/Product/src/Repositories/ProductInventoryRepository.php

```diff
@@ -37,4 +39,18 @@ public function saveInventories(array $data, $product)
             ]);
         }
     }
-}
\ No newline at end of file
+
+    /**
+     * @return Collection
+     */
+    public function getStockThreshold(): Collection
+    {
+        return $this->getModel()
+            ->with('product', 'product.attribute_family', 'product.attribute_values', 'product.images')
+            ->select('product_id', DB::raw('SUM(qty) as total_qty'))
```

Comment:
Whenever using DB raw use table prefix also.

Reply:
Thank you for your comment. In this specific query, I'm using a raw expression to sum the 'qty' column. Since I'm not directly referencing any table name in the raw query, there's no need to include a table prefix here. The table prefix is only necessary when we use raw SQL expressions that include table names.

Reply:
Thank you for your comment. In this specific query, I'm using a raw expression to sum the 'qty' column. Since I'm not directly referencing any table name in the raw query, there's no need to include a table prefix here. The table prefix is only necessary when we use raw SQL expressions that include table names.

Reply:
Okay.